src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -133,36 +155,46 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
             placeholder="Password"
             ref={register({
               required: true,
-              // minLength: 8,
-              pattern: /^(?=.*[0-9])(?=.*[a-zA-Z]).{2,}$/,
+              pattern: /^(?=.*[0-9])(?=.*[a-zA-Z]).{6,}$/,
             })}
           />
+
           <span
             className={`input-${
               errors.password && errors.password.type === "pattern"
                 ? "error"
                 : "info"
             }`}
           >
-            Password must contain letters and numbers
+            Password must contain letters, numbers, and be at least 6 characters
+            long
```

Comment:
Note: this was only checking for length 2, whereas the backend was checking for length 6. The discrepancy between frontend/backend validation rules was what lead to the issue as described in the original ticket showing up as it did, though the main bug that was fixed is as described in the PR description.

Reply:
Yep, spotted this in review, appreciate the change to 6 characters.